irohad/ametsuchi/impl/postgres_wsv_query.cpp

```diff
@@ -124,6 +124,9 @@ namespace iroha {
       processSoci(st, ind, row, [&set](std::string &row) {
         set = shared_model::interface::RolePermissionSet(row);
       });
+      if (set.none()) {
```

Comment:
Isn't it IR-1480?

Reply:
It looks like the task is outdated :pensive: Since roles with no permissions are allowed at the moment, `boost::none` can be returned only as a result of database error.